compiler/src/main/scala/dx/compiler/ExecutableCompiler.scala

```diff
@@ -297,6 +297,17 @@ class ExecutableCompiler(extras: Option[Extras],
     }
   }
 
+  // Merge initial description & extended description
+  private def descriptionToNative(description: Option[String],
+                                  extendedDescription: Option[String]): Map[String, JsValue] = {
+    (description, extendedDescription) match {
+      case (Some(d), Some(e)) => Map("description" -> JsString(d + "\n\n" + e))
```

Comment:
Use format strings rather than concatenation. `s"${d}\n\n${e}"`

Reply:
You can also do

```scala
Map("description" -> JsString(Vector(description, extendedDescription).flatten.mkString("\n\n")))
```

Reply:
Took your 2nd suggestion.